shell/platform/windows/platform_handler.h

```diff
@@ -31,6 +31,11 @@ class PlatformHandler {
       std::unique_ptr<MethodResult<rapidjson::Document>> result,
       std::string_view key) = 0;
 
+  // Provides a boolean to |result| as the value in a dictionary at key
+  // "value" representing whether or not the clipboard has a non-empty string.
+  virtual void HasStrings(
```

Comment:
Sorry if I'm revisiting an established design. But this name looks pretty confusing. Why is it `HasString` instead of `HasString`? And can we rename it to a "verb", such as `GetHasString` to represent the fact that it is a query from the framework?

Reply:
Plural `HasStrings` came from native iOS [hasStrings](https://developer.apple.com/documentation/uikit/uipasteboard/1829416-hasstrings?language=objc), which is where all of this started.  I'll change the Windows methods to `GetHasStrings` while keeping the method channel name the same "Clipboard.hasStrings".